### PR TITLE
Added input box to character count demo

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**
Also did some minor cleanup to the demo code

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-demo-input-box](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-demo-input-box)

File: src/app/public/plugin-resources/code-examples/character-count/character-count-demo.module.ts [hunk 1]
```diff
@@ -34,5 +41,4 @@ import {
   ]
 })
 
-export class SkyRadioDemoModule {
-}
+export class SkyCharacterCountDemoModule { }
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/character-count/character-count-demo.module.ts:44]: Nice catch.

**Review by Blackbaud-AlexKingman — APPROVED** [review]
Looks good.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-demo-input-box](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-character-count-demo-input-box)

**Pull request merged**